swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory is None:
+            inventory = []
+        self.inventory = inventory
```

Comment:
Love this validation check from the beginning! For fun, we can swap this out for a one-line ternary, if you wanted:
```
self.inventory = [] if inventory is None else inventory
```